includes/Helper/ExportRow.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\WooCommerce\Helper;
+
+use ErrorException;
+use MyParcelNL\Sdk\src\Model\Consignment\AbstractConsignment;
+use WC_Order;
+use WC_Order_Item_Product;
+use WC_Product;
+use WCMP_Export;
+use WCMYPA_Admin;
+use WCMYPA_Settings;
+use WPO\WC\MyParcel\Compatibility\Product as WCX_Product;
+
+class ExportRow
+{
+    public const DEFAULT_PRODUCT_QUANTITY = 1;
+    public const CURRENCY_EURO = 'EUR';
+
+    /**
+     * @var \WC_Order
+     */
+    private $order;
+
+    /**
+     * @var \WC_Product
+     */
+    private $product;
+
+    public function __construct(WC_Order $order, WC_Product $product)
+    {
+        $this->order   = $order;
+        $this->product = $product;
+    }
+
+    /**
+     * @return string
+     * @throws \JsonException
+     */
+    public function getCountryOfOrigin(): string
+    {
+        $defaultCountryOfOrigin   = WCMYPA()->setting_collection->getByName(WCMYPA_Settings::SETTING_COUNTRY_OF_ORIGIN);
+        $productCountryOfOrigin   = WCX_Product::get_meta($this->product, WCMYPA_Admin::META_COUNTRY_OF_ORIGIN, true);
+        $variationCountryOfOrigin = WCX_Product::get_meta(
+            $this->product,
+            WCMYPA_Admin::META_COUNTRY_OF_ORIGIN_VARIATION,
+            true
+        );
+        $fallbackCountryOfOrigin  = WC()->countries->get_base_country() ?? AbstractConsignment::CC_NL;
+
+        return $variationCountryOfOrigin ?: $productCountryOfOrigin ?: $defaultCountryOfOrigin ?: $fallbackCountryOfOrigin;
+    }
+
+    /**
+     * @return int
+     * @throws \ErrorException|\JsonException
+     */
+    public function getHsCode(): int
+    {
+        $defaultHsCode   = WCMYPA()->setting_collection->getByName(WCMYPA_Settings::SETTING_HS_CODE);
+        $productHsCode   = WCX_Product::get_meta($this->product, WCMYPA_Admin::META_HS_CODE, true);
+        $variationHsCode = WCX_Product::get_meta($this->product, WCMYPA_Admin::META_HS_CODE_VARIATION, true);
+
+        $hsCode = $productHsCode ?: $defaultHsCode;
```

Comment:
Is het misschien mooier om gewoon van deze regel $hsCode = $variationHsCode ?: $productHsCode ?: $defaultHsCode; te maken?

Reply:
Inderdaad, dat lijkt mij ook dat dat hetzelfde is? Of kan er `null` uit `::get_meta` komen?

Reply:
Je kan uit de `::get_meta` null terug krijgen (het kan eigenlijk alles zijn ` * @return mixed`)